go/cli/internal/frameworks/adk/python/templates/Dockerfile.tmpl

```diff
@@ -0,0 +1,15 @@
+ARG DOCKER_REGISTRY=ghcr.io
+# TODO: use latest tag once it is available
+ARG VERSION=0.6.7
```

Comment:
This is currently hardcoded to the latest version as we do not publish the `kagent-adk` image with the tag `latest`. This should be updated once we do publish an image with the tag `latest`

Reply:
Can you create an issue to tag this with latest.

Reply:
created an issue [here](https://github.com/kagent-dev/kagent/issues/845)